Sources/StreamChat/Workers/UserListUpdater.swift

```diff
@@ -66,9 +72,11 @@ enum UpdatePolicy {
     case merge
     /// The resulting user set of the query will replace the existing user set.
     case replace
+    /// The resulting user set of the query is not stored locally.
+    case ephermal
```

Comment:
Not sure if `UpdatePolicy` would be still the correct wording here. Basically, `ephermal`, as an update policy doesn't sound correct, wdyt? 
Another point, there's a typo here, should be `ephemeral` 😁

Reply:
I would remove the UpdatePolicy all thogether. Search should hit the server, and never go through the DB. The results should always reflect the server

Reply:
logically, that makes sense. But, we need to check if it's used somewhere else or breaks some existing logic.

Reply:
Removed the added policy